docs/Doxyfile.config

```diff
@@ -913,7 +913,8 @@ RECURSIVE              = YES
 
 EXCLUDE                = "@OpenVINO_SOURCE_DIR@/thirdparty" \
                          "@OpenVINO_SOURCE_DIR@/temp" \
-                         "@OpenVINO_SOURCE_DIR@/bin"
+                         "@OpenVINO_SOURCE_DIR@/bin" \
+                         "@OpenVINO_SOURCE_DIR@/src/core/dev_api/ops/docs"
```

Comment:
Documentation CI check is failing because of
`doxygen_errors = ["document isn't included in any toctree"]`
This change was a suggestion from @kblaszczak-intel to avoid such issue and exclude it from the build, but it is still failing with the same error.

Reply:
Removal of the tags in the spec files fixed this issue, Documentation CI passed (commit https://github.com/openvinotoolkit/openvino/pull/12162/commits/63244083c69414d833575a10cb16a7ec7004b898)